main/Accordion/AccordionItem.tsx

```diff
@@ -191,30 +182,27 @@ const AccordionItem: FC<Props> = (props) => {
         testID={`title_${testID}`}
         onPress={handlePress}
         activeOpacity={activeOpacity}
-        style={styles?.titleContainer}
+        style={titleContainer}
       >
         {renderTitle(item.title)}
         {renderIndicator(toggleElement)}
       </TitleContainer>
 
-      <Animated.View
-        testID={`body_${testID}`}
-        style={{
-          height: bodyMounted ? bodyHeight : undefined,
-          transform: [
-            {
-              translateY: dropDownAnimValueList,
-            },
-          ],
-        }}
-        onLayout={handleBodyLayout}
-      >
-        {item.bodies.map((body, key) => (
-          <ItemContainer key={key} theme={theme} style={styles?.bodyContainer}>
-            {renderBody(body)}
-          </ItemContainer>
-        ))}
-      </Animated.View>
+      {
```

Comment:
Why are you wrapping an Animated.View  with curly braces?

Reply:
It's my mistake. i put the conditional statement and erased it. thanks!